### PR TITLE
Implementation of the stop locator plugin

### PR DESCRIPTION
**Pull request by briansetzesglobant**
The functionalities of returning the last coordinates of the user were implemented, and also stopping the locator

Video
https://user-images.githubusercontent.com/84338449/127376727-0ee0f855-b19a-4768-8754-d5124dfd1b57.mp4

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -149,6 +169,57 @@ class _MyAppState extends State<MyApp> {
...
+                          _coordinates = snapshot.data;
+                          if (!_coordinates.containsKey(Strings.status)) {
+                            _latitude = _coordinates[Strings.latitude]
+                                .toStringAsFixed(Numbers.numberCoordinates);
+                            _longitude = _coordinates[Strings.longitude]
+                                .toStringAsFixed(Numbers.numberCoordinates);
+                          } else {
+                            _latitude = Strings.noLatitude;
+                            _longitude = Strings.noLongitude;
+                          }
```
Comment (FranciscoGaraventa) [inline example/lib/main.dart]: You have a lot of logic here. An option is to create a class that hold latitude and longitude, and a method that set this two values as parameter of this method is snapshot.data

Comment (FranciscoGaraventa) [reply example/lib/main.dart]: snapshot.data != null can be replace with snapshot.hasData?

File: example/lib/main.dart [hunk 2]
```diff
@@ -20,6 +21,9 @@ class _MyAppState extends State<MyApp> {
   String _platformVersion = Strings.valueUnknown;
   bool _initializeLocatorPlugin = false;
   bool _checkPermissions = false;
+  var _latitude = Strings.noLatitude;
+  var _longitude = Strings.noLongitude;
+  Map<dynamic, dynamic> _coordinates = {};
```
Comment (FranciscoGaraventa) [inline example/lib/main.dart]: _coordinates can be deleted from this scope, and latitude/longitude will be inside the new class that I explain in the comment below

File: example/lib/main.dart (hunk 1 above)
Comment (briansetzesglobant) [reply example/lib/main.dart]: done

File: example/lib/main.dart (hunk 2 above)
Comment (briansetzesglobant) [reply example/lib/main.dart]: done

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Pull request merged**